apis/project.cattle.io/v3/schema/schema.go

```diff
@@ -1020,3 +1022,70 @@ func monitoringTypes(schemas *types.Schemas) *types.Schemas {
 		).
 		MustImport(&Version, monitoringv1.Alertmanager{}, projectOverride{})
 }
+
+func autoscalingTypes(schemas *types.Schemas) *types.Schemas {
+	return schemas.
+		AddMapperForType(&Version, autoscaling.HorizontalPodAutoscaler{},
+			&m.ChangeType{Field: "scaleTargetRef", Type: "reference[workload]"},
+			&m.Move{From: "scaleTargetRef", To: "workloadId"},
+			mapper.CrossVersionObjectToWorkload{Field: "workloadId"},
+			&m.Required{Fields: []string{"workloadId", "maxReplicas"}},
+			&m.AnnotationField{Field: "displayName"},
+			&m.DisplayName{},
+			&m.AnnotationField{Field: "description"},
+			&m.Embed{Field: "status"},
+			mapper.NewMergeListByIndexMapper("currentMetrics", "metrics", "type"),
+		).
+		AddMapperForType(&Version, autoscaling.MetricTarget{},
+			&mapper.IntOrStringParser{Field: "averageUtilization"},
```

Comment:
Why do we need the `IntOrStringParser` given that `averageUtilization` field is int